doc/install_windows.txt

```diff
@@ -1,633 +1,125 @@
 
-  git clone https://github.com/Theano/Theano.git --branch rel-0.7
-
-Either way, a folder `Theano` will be created with the library
-downloaded to it.
-
-
-Manual Installation
-###################
-
-To get the latest bleeding edge version got to `Theano on GitHub
-<https://github.com/Theano/Theano>`_ and download the `latest zip
-<https://github.com/Theano/Theano/archive/master.zip>`_. Then unzip it
-somewhere.
-
-Alternatively, you can check the latest release release 0.7 (as of March
-2015) by going to
-https://github.com/Theano/Theano/releases/tag/rel-0.7 and
-downloading the `zip
-<https://github.com/Theano/Theano/archive/rel-0.7.zip>`_.
-
-Configuring Theano
-##################
-
-Once you have installed Theano, open the Python Shell
-(e.g. ``c:\scisoft\env.bat`` if you follow the installation directories
-from this tutorial) and ``cd`` to it. Then run::
-
-  python setup.py develop
-
-this step will add the Theano directory to you ``PYTHON_PATH``
-environment variable.
-
-At this stage you can check whether Theano works and is able to
-compile C code for CPU execution.
-
-Create a test file containing:
-
-.. testcode::
-
-   import numpy as np
-   import time
-   import theano
-   A = np.random.rand(1000,10000).astype(theano.config.floatX)
-   B = np.random.rand(10000,1000).astype(theano.config.floatX)
-   np_start = time.time()
-   AB = A.dot(B)
-   np_end = time.time()
-   X,Y = theano.tensor.matrices('XY')
-   mf = theano.function([X,Y],X.dot(Y))
-   t_start = time.time()
-   tAB = mf(A,B)
-   t_end = time.time()
-   print("NP time: %f[s], theano time: %f[s] (times should be close when run on CPU!)" %(
-                                              np_end-np_start, t_end-t_start))
-   print("Result difference: %f" % (np.abs(AB-tAB).max(), ))
-
-.. testoutput::
-   :hide:
-   :options: +ELLIPSIS
-
-   NP time: ...[s], theano time: ...[s] (times should be close when run on CPU!)
-   Result difference: ...
-
-.. code-block:: none
-
-   NP time: 1.480863[s], theano time: 1.475381[s] (times should be close when run on CPU!)
-   Result difference: 0.000000
-
-Then run it. It should execute without problems and the Theano function
-should run at a speed similar to the regular NumPy
-multiplication. (Both Numpy and Theano should call the same BLAS
-routine for matrix multiplication)
-
-
-Configure Theano for GPU use
-############################
-
-Install `libgpuarray <http://deeplearning.net/software/libgpuarray/installation.html>`_ if you have not already done so.
-
-Theano can be configured with a ``.theanorc`` text file (or
-``.theanorc.txt``, whichever is easier for you to create under
-Windows). It should be placed in the directory pointed to by the
-``%USERPROFILE%`` variable. Please note, that WinPython changes it to
-``WinPythonDir\settings`` (so in our system this corresponds to
-``c:\scisoft\WinPython-64bit-2.7.9.4\settings``.
-
-To use the GPU please write the following configuration file:
+    pip install pydot-ng
```

Comment:
I think you can do as for Python 2 about pydot-ng.

In fact, I don't see a difference anymore between Python 2 and 3